index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
         <section class="social-details">
           <h5>LET'S CONNECT</h5>
           <div class="social-wrapper">
-            <span class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></span>
-            <span class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></span>
-            <span class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></span>
+            <ul>
+              <li class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></li>
+              <li class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></li>
+              <li class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></li>
+            </ul>
```

Comment:
- [OPTIONAL] Good job on adding icons but kindly add links to icons to redirect to your social medias. 👍